progressReport/templates/web_ui.html

```diff
@@ -163,7 +163,7 @@
             root.y0 = width / 2;
             assignStudentLevels(root);
             update(root);
-            d3.select(self.frameElement).style("height", "100%");
+            d3.select(self.frameElement).style("height", "500px");
```

Comment:
You cannot change this value.  It has to be 100% for the main version to work.  It looks like you may not have pulled the changes from main before making these changes, so before working on anything else, make sure your repo is up to date with main.